components/builder-minio/habitat/default.toml

```diff
@@ -35,8 +35,14 @@ bind_port = 9000
 # Setup additional environment variables to pass to minio
 # Doc: https://docs.min.io/docs/minio-server-configuration-guide.html
 [env]
+MINIO_ENDPOINT="http://localhost:9000"
+MINIO_BUCKET="habitat-builder-artifact-store.local"
```

Comment:
see above. You may not need these two.

Reply:
I think in light of the user.toml files and the default values of bldr.env.sample this is probably the way to go.  However, all three of these probably break the same.

Reply:
Done.